src/libinjection.h

```diff
@@ -24,6 +24,7 @@ LIBINJECTION_BEGIN_DECLS
  * Pull in size_t
  */
 #include <string.h>
+#include "libinjection_html5.h"
```

Comment:
I think we should move the typedef to this file and include it in others instead.